src/main/java/com/tvtracker/enterprise/dao/BaseDAO.java

```diff
@@ -120,8 +120,9 @@ public void addWhere(String column, int value) {
      * @param value value to be populated into the given column
      */
     public void setColumnValue(String column, Object value) {
-        if(columnValues == null)
+        if(columnValues == null) {
             columnValues = new HashMap<>();
+        }
```

Comment:
Adding If brackets for clarity. 

Source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3